pkg/cmd/ssh/options.go

```diff
@@ -311,6 +317,12 @@ func (o *SSHOptions) Complete(f util.Factory, cmd *cobra.Command, args []string)
 		o.BastionName = name
 	}
 
+	if o.SSHUser == "" {
+		SSHNodeUsername = DefaultUsername
+	} else {
+		SSHNodeUsername = o.SSHUser
+	}
+
```

Comment:
initialize the default here:

```go
// NewSSHOptions returns initialized SSHOptions.
func NewSSHOptions(ioStreams util.IOStreams) *SSHOptions {
	return &SSHOptions{
		// .....
		SSHUser:               DefaultUsername,
	}
}
```
Then in `func (o *SSHOptions) Validate() error`, validate that a user is set

```go
func (o *SSHOptions) Validate() error {
	// .....
	if o.SSHUser == "" {
		return errors.New("user must not be empty")
	}
```

Reply:
fff5ed4df8cbd95f476ef1e927256d5f33642552